src/python/pants/init/target_roots_calculator.py

```diff
@@ -127,23 +127,38 @@ def create(cls, options, session, symbol_table, build_root=None):
     # initialization paths.
     changed_options = options.for_scope('changed')
     changed_request = ChangedRequest.from_options(changed_options)
-
+    owned_files = options.for_global_scope().owner_of
     logger.debug('spec_roots are: %s', spec_roots)
     logger.debug('changed_request is: %s', changed_request)
     scm = get_scm()
     change_calculator = ChangeCalculator(session, symbol_table, scm) if scm else None
+    owner_calculator = OwnerCalculator(session, symbol_table) if owned_files else None
+    logger.debug('owner_files are: %s', owned_files)
+    targets_specified = [1 for spec
+                         in (changed_request.is_actionable(), owned_files, spec_roots)
+                         if spec]
+
+    if len(targets_specified) > 1:
+      # We've been provided a more than one of: a change request, an owner request, or spec roots.
+      raise InvalidSpecConstraint(
+        'Only one target specification can be provided out of these three options: '
+        '--changed-*, --owner-of=, or a specific target'
```

Comment:
would `s/or a specific target/or target specs/` - since there can be 1) multiple targets passed 2) specs that expand to multiple targets.

Reply:
Yep, didn't want to use 'specs' because I wasn't sure if it was okay to abbreviate. But you're right it should be plural.